client/src/footer.js

```diff
@@ -0,0 +1,17 @@
+// import React from 'react';
+// import './App.css';
+
+
+// function Footer() {
+//     return (
+//       <div>
+//         <h1>Foot</h1>
+//       </div>
+//     );
+//   }
+
+
+
+
+
+// export default Footer;
```

Comment:
We can just remove this file since your Navbar and Footer are just coded right into your App.js